lib/matplotlib/axes/_axes.py

```diff
@@ -3876,15 +3878,15 @@ def scatter(self, x, y, s=None, c=None, marker=None, cmap=None, norm=None,
             except ValueError:
                 raise ValueError("'color' kwarg must be an mpl color"
                                  " spec or sequence of color specs.\n"
-                                 "For a sequence of values to be"
-                                 " color-mapped, use the 'c' kwarg instead.")
+                                 "For a sequence of values to be color-mapped,"
+                                 " use the 'c' named argument instead.")
```

Comment:
I don't think there is a strict distinction in the terminology between keyword arguments and named arguments. I agree that it's unfortunate to use "kwarg" here because that's the name of the variable keyword argument list.

The technical details are irrelevant here. I'd just go for "use the argument 'c' instead."

Reply:
See 613b5aa :) .

BTW, if there is a native English-speaker around, would they mind answering a question that often bothers me when editing docstrings ^^ : is it better to say “the *x* argument” or “the argument *x*”?